build_deploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -exv
+
+IMAGE="quay.io/cloudservices/insights-inventory-frontend"
```

Comment:
You can read the app name from `package.json#insights.appname` this way we could come up with a clever idea to generate this on the fly and apps won't have to change anything in this file.